meetings/2020-12-15.md

```diff
@@ -0,0 +1,58 @@
+# ISIS TC Meeting December 15, 2020 @11:30AM MST
+
+## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+
+### Attending
+
+-
+
+## Agenda / Notes
+
+- Action Items from last meeting
+  - LROC release stories
+  - Kris release stories
+  - TSC application governance update
+
+- Release Working Group
+
+- Collecting Epic Changes
+  - How do we propose, discuss, and evaluate large scale changes to ISIS?
+    - Examples of such changes:
+      - redoing the ISIS website and documentation
+      - Creating a C interface for ISIS
+      - Separating the Library and applications
+      - Introducing a deprecation policy
+  - Currently:
+    - RFCs
+      - Wiki page with accompanying issue
+      - Done before work starts
+      - Accept/reject mechanism is unclear
+    - Pull Requests
+      - At what point does a pull request need an RFC?
+      - What about feature branches?
+      - Work has to be completed for discussion to start
+  - How do other software projects handle this?
+    - OpenCV uses [Evolution Proposals](https://github.com/opencv/opencv/wiki/Evolution-Proposals)
+    - Python uses [PEPs](https://www.python.org/dev/peps/pep-0001/)
+  - What do we want out of such a process?
+
+## Discussions for next meeting
+
+- Action Items from last meeting
+
+## Action Items
+
+- Victor
+  - LROC meeting the 11th, user & dev stories to contribute by next meeting
+- Kris
+  - Submit stories for release working group
+- Jesse
+  - Kick off release WG
+  - TSC application governance language, update app to reflect reality.
+  - Provide more context for collecting "epic" changes
+
+
+## Next Meeting
+
+- January 12th @ 12pm MST
```

Comment:
Should we start with a different preset time? I think this conflicts with OpenPlanetary lunch.